.gitignore

```diff
@@ -1,38 +1,30 @@
-# Build files
-pom.xml
-*jar
-/lib/
-/classes/
+
+# Leiningen
 /target/
-/docs/
-/pkg/
-/log/
-*.tar.gz
-version
-junit
-
-# Emacs files
+
+# Emacs
 *#
+*~
+.#*
 
 # Local overide for testing config
-test-resources/config/local.clj
+/test-resources/config/local.clj
 
-# Vagrant cache folder & vagrantfile if used
-.vagrant
-Vagrantfile
+# Vagrant
+/.vagrant/
```

Comment:
If this is too conservative, (now or later) we could add other likely places, or we could add some inverted exclusions like "not in src", or we could go back to a free-floating `.vagrant/`.